#
# Submitted-By: Marc Kleine-Budde, 2006-12-25
# Committed-By: Marc Kleine-Budde
#
# Error:
#
# python install is called w/o giving a prefix
#
# Description:
#
# This patch add --prefix to python install
#
# State:
#
# should be send upstream
#
Index: ipkg-utils-050831/Makefile
===================================================================
--- ipkg-utils-050831.orig/Makefile
+++ ipkg-utils-050831/Makefile
@@ -12,7 +12,7 @@ ipkg-compare-versions: ipkg-compare-vers
 
 install: ${UTILS}
 	cp ${UTILS} $(PREFIX)/bin
-	python setup.py install
+	python setup.py install --prefix=$(PREFIX)
 	chmod agu+rx ipkg-make-index
 	cp -f ipkg-make-index $(PREFIX)/bin
 
